src/main/webapp/dev/js/main/feedbackSubmissionsEdit.es6

```diff
@@ -919,6 +923,15 @@ function hasWarningMessage() {
     return $(WARNING_STATUS_MESSAGE).length;
 }
 
+function isSessionClosingSoon() {
+    const endDate = new Date($(END_TIME).html().trim());
```

Comment:
Will add this information to the page in #7736 and then handle in time zone here.